.ci/docker-compose-file/docker-compose-kafka.yaml

```diff
@@ -10,7 +10,9 @@ services:
     networks:
       emqx_bridge:
   ssl_cert_gen:
-    image: fredrikhgrelland/alpine-jdk11-openssl
+    build:
+      context: ./kafka
+      dockerfile: ssl_cert_gen.Dockerfile
```

Comment:
the image above is not multi-platform, so if you try to run kafka bridge tests on e.g. apple silicon, you gonna have some fun trying to get it working
on the other hand, the image is simple enough and the build does not take much longer than pulling it